main.tf

```diff
@@ -83,4 +85,6 @@ module "snowflake_custom_role" {
       object_name       = snowflake_resource_monitor.this.name
     }]
   }
+
+  depends_on = [snowflake_resource_monitor.this]
```

Comment:
do we need this since above there's a dependency on resource monitor via output value?

Reply:
Removed - you're right